CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
    * FIXED: Use kServiceRoad edges while searching for ferry connection [#2933](https://github.com/valhalla/valhalla/pull/2933)
    * FIXED: Enhanced logic for IsTurnChannelManeuverCombinable [#2952](https://github.com/valhalla/valhalla/pull/2952)
    * FIXED: Restore compatibility with gcc 6.3.0, libprotobuf 3.0.0, boost v1.62.0 [#2953](https://github.com/valhalla/valhalla/pull/2953)
+   * FIXED: Removed unexpected maneuvers at sharp bends [#2968](https://github.com/valhalla/valhalla/pull/2968)
```

Comment:
This PR has less to do with sharpness than with excluding track use roads for maneuvers, right?
```suggestion
   * FIXED: Removed unexpected maneuvers at track roads [#2968](https://github.com/valhalla/valhalla/pull/2968)
```

Reply:
yes and no - more than just tracks - other different travel modes too
All of the issues were at sharp bends

Reply:
I see the call to `is_forward` now, I'd missed that. 👍